requirements/requirements-py-3.9.txt

```diff
@@ -103,6 +103,7 @@ pytest-benchmark==3.2.3
 pytest-cov==2.10.1
 pytest-rerunfailures==9.1.1
 pytest-timeout==1.4.2
+pytest-asyncio==0.12.0
```

Comment:
Is this dependency specific to 3.9? If yes, then this should be indicated via markers in setup.json. Otherwise, I'd suggest we add this to all requirements files.

Reply:
This _is_ present in all requirement files. They were added I imagine because `pytest-ascynio` is a dependency of one of our dependencies that installs it for testing. Since now we add it as our own dependency it also ends up in the Python 3.9 requirements. Not 100% sure why it wasn't there yet.

Reply:
Ah, sorry, I didn't check the other files.